src/js/components/Image/Image.js

```diff
@@ -2,7 +2,19 @@ import React, { forwardRef, useState } from 'react';
 import { StyledImage } from './StyledImage';
 
 const Image = forwardRef(
-  ({ a11yTitle, fallback, onError, opacity, fill, src, ...rest }, ref) => {
+  (
+    {
+      a11yTitle,
+      'aria-label': ariaLabel,
```

Comment:
I think the `aria-label` will work transparently via the `...rest` spread. So I don't think we need explicit attention for that.